### DISCUSSION TITLE
Preinstall tool failed, please try again

### DISCUSSION THREAD

**Discussion by Yellster** [he3-app/he3-feedback#220] [Bug Report]
你好，我从官网下载安装包安装上之后，预安装一直失败
我的操作系统为，Windows 10 企业版 21H2 19044.2965

![Snipaste_2023-06-09_16-31-37](https://github.com/he3-app/he3-feedback/assets/37898221/5a971982-de45-47a5-9cc2-2ada3132e0e6)

**Comment by lyzhang1999** [comment]
请检查网络对以下域名的访问情况：

api.he3app.com
cdn.he3app.com
portal.he3app.com
t.he3app.com
docs.he3app.com
webview.he3app.com

**Reply by Yellster** [reply]
网络都是通的 😅

**Comment by atomicmm** [comment]
me 2, 操作系统是 ubuntu 22，旧版本可用，新版本（1.3.17）打开一直报这个错

**Reply by lyzhang1999** [reply]
检查一下这几个域名都通吗？

api.he3app.com
cdn.he3app.com
portal.he3app.com
t.he3app.com
docs.he3app.com
webview.he3app.com

**Reply by Yellster** [reply]
请问旧版本哪里可以下载呢

**Comment by zzhzero** [comment]
最新可离线使用的是哪个版本

**Comment by lyzhang1999** [comment]
我们晚点会发布离线版的能力，到时候会在公众号「开发者工具推荐」上发送推文。
